src/device/pf_port.c

```diff
@@ -225,15 +225,23 @@ pf_mediatype_values_t pf_port_get_media_type (pnal_eth_mau_t mau_type)
    case PNAL_ETH_MAU_RADIO:
       return PF_PD_MEDIATYPE_RADIO;
    case PNAL_ETH_MAU_COPPER_10BaseT:
+   case PNAL_ETH_MAU_COPPER_10BaseTX_HALF_DUPLEX:
+   case PNAL_ETH_MAU_COPPER_10BaseTX_FULL_DUPLEX:
    case PNAL_ETH_MAU_COPPER_100BaseTX_HALF_DUPLEX:
    case PNAL_ETH_MAU_COPPER_100BaseTX_FULL_DUPLEX:
    case PNAL_ETH_MAU_COPPER_1000BaseT_HALF_DUPLEX:
    case PNAL_ETH_MAU_COPPER_1000BaseT_FULL_DUPLEX:
+   case PNAL_ETH_MAU_COPPER_10000BaseT:
+   case PNAL_ETH_MAU_COPPER_5000BaseT:
+   case PNAL_ETH_MAU_COPPER_2500BaseT:
       return PF_PD_MEDIATYPE_COPPER;
    case PNAL_ETH_MAU_FIBER_100BaseFX_HALF_DUPLEX:
    case PNAL_ETH_MAU_FIBER_100BaseFX_FULL_DUPLEX:
    case PNAL_ETH_MAU_FIBER_1000BaseX_HALF_DUPLEX:
    case PNAL_ETH_MAU_FIBER_1000BaseX_FULL_DUPLEX:
+   case PNAL_ETH_MAU_FIBER_10000BaseR:
+   case PNAL_ETH_MAU_FIBER_2500BaseX:
+   case PNAL_ETH_MAU_FIBER_5000BaseR:
```

Comment:
This is a *super minor* detail, but maybe we should order all these lists according to size/speed?  Like you did below.

Reply:
Earlier they were sorted according to the bit positions, but I agree, it is more readable when they sorted according to the speed. I'll sort them everywhere according to the speed.

Reply:
Great, I agree!